tests/main/hooks-env/task.yaml

```diff
@@ -0,0 +1,16 @@
+summary: Check that environment variables are interpolated when running hooks.
+
+prepare: |
+    . "$TESTSLIB/snaps.sh"
+    install_local snapctl-hooks
+
+environment:
+    ENVDUMP: /var/snap/snapctl-hooks/current/hooks-env
+
+execute: |
+    snap set snapctl-hooks command=dump-env
+
+    echo "Test that environment variables were interpolated"
+    cat $ENVDUMP | MATCH "^TEST_COMMON=/var/snap/snapctl-hooks/common$"
+    cat $ENVDUMP | MATCH "^TEST_DATA=/var/snap/snapctl-hooks/.*$"
+    cat $ENVDUMP | MATCH "^TEST_SNAP=/snap/snapctl-hooks/.*$"
```

Comment:
Test looks nice, but can we please move this into snap-run-hook which is where we already test the basic hook running logic?